src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2858,10 +2858,11 @@ static WellKnownMembers()
                 (byte)(MemberFlags.Method | MemberFlags.Static),                                                                                    // Flags
                 (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_CodeAnalysis_Runtime_Instrumentation - WellKnownType.ExtSentinel),  // DeclaringTypeId
                 0,                                                                                                                                  // Arity
-                    4,                                                                                                                              // Method Signature
+                    5,                                                                                                                              // Method Signature
```

Comment:
Exposing a version number from the runtime is an interesting question. My first guess is that the compiler should infer the version based on the presence/absence of runtime members. Should we be prepared to deal with runtime semantic changes that don't involve any signature changes?